App.js

```diff
@@ -1,24 +1,27 @@
-import React from 'react'
+import React, { useState } from 'react'
 import styled from 'styled-components/native'
+import background from './assets/background.jpg'
```

Comment:
Next time let's remember about line empty line space between outer and local dependencies